build.sh

```diff
@@ -48,13 +48,11 @@ go generate ./...
 
 function install {
   if [ "${1}" != 'native' ]; then
-    GOOS="${1}" GOARCH="${2}" CGO_ENABLED=0 go get -ldflags "-X main.revision=$(git rev-parse HEAD)" -v ./...
```

Comment:
Same here, let's use `go install` so we get the binaries built under `"${GOPATH}/bin"`.

```
GOOS="${1}" GOARCH="${2}" CGO_ENABLED=0 go install -ldflags "-X main.revision=$(git rev-parse HEAD)" -v ./...
```